{'name': 'root', 'msg': 'Mensaje a nivel debug', 'args': (), 'levelname': 'DEBUG', 'levelno': 10, 'pathname': 'C:\\Users\\Usuario\\Desktop\\ClicOH\\Programacion\\FastAPI\\Prueba-CRUD\\logger.py', 'filename': 'logger.py', 'module': 'logger', 'exc_info': None, 'exc_text': None, 'stack_info': None, 'lineno': 14, 'funcName': '<module>', 'created': 1679061949.703206, 'msecs': 703.0, 'relativeCreated': 2.518892288208008, 'thread': 31644, 'threadName': 'MainThread', 'processName': 'MainProcess', 'process': 29348, 'message': 'Mensaje a nivel debug'}sc(time)s: DEBUG [logger.py: 14] Mensaje a nivel debug
{'name': 'root', 'msg': 'Mensaje a nivel info', 'args': (), 'levelname': 'INFO', 'levelno': 20, 'pathname': 'C:\\Users\\Usuario\\Desktop\\ClicOH\\Programacion\\FastAPI\\Prueba-CRUD\\logger.py', 'filename': 'logger.py', 'module': 'logger', 'exc_info': None, 'exc_text': None, 'stack_info': None, 'lineno': 15, 'funcName': '<module>', 'created': 1679061949.703206, 'msecs': 703.0, 'relativeCreated': 2.518892288208008, 'thread': 31644, 'threadName': 'MainThread', 'processName': 'MainProcess', 'process': 29348, 'message': 'Mensaje a nivel info'}sc(time)s: INFO [logger.py: 15] Mensaje a nivel info
{'name': 'root', 'msg': 'Mensaje a nivel warning', 'args': (), 'levelname': 'WARNING', 'levelno': 30, 'pathname': 'C:\\Users\\Usuario\\Desktop\\ClicOH\\Programacion\\FastAPI\\Prueba-CRUD\\logger.py', 'filename': 'logger.py', 'module': 'logger', 'exc_info': None, 'exc_text': None, 'stack_info': None, 'lineno': 16, 'funcName': '<module>', 'created': 1679061949.703206, 'msecs': 703.0, 'relativeCreated': 2.518892288208008, 'thread': 31644, 'threadName': 'MainThread', 'processName': 'MainProcess', 'process': 29348, 'message': 'Mensaje a nivel warning'}sc(time)s: WARNING [logger.py: 16] Mensaje a nivel warning
{'name': 'root', 'msg': 'Mensaje a nivel error', 'args': (), 'levelname': 'ERROR', 'levelno': 40, 'pathname': 'C:\\Users\\Usuario\\Desktop\\ClicOH\\Programacion\\FastAPI\\Prueba-CRUD\\logger.py', 'filename': 'logger.py', 'module': 'logger', 'exc_info': None, 'exc_text': None, 'stack_info': None, 'lineno': 17, 'funcName': '<module>', 'created': 1679061949.703206, 'msecs': 703.0, 'relativeCreated': 2.518892288208008, 'thread': 31644, 'threadName': 'MainThread', 'processName': 'MainProcess', 'process': 29348, 'message': 'Mensaje a nivel error'}sc(time)s: ERROR [logger.py: 17] Mensaje a nivel error
{'name': 'root', 'msg': 'Mensaje a nivel critical', 'args': (), 'levelname': 'CRITICAL', 'levelno': 50, 'pathname': 'C:\\Users\\Usuario\\Desktop\\ClicOH\\Programacion\\FastAPI\\Prueba-CRUD\\logger.py', 'filename': 'logger.py', 'module': 'logger', 'exc_info': None, 'exc_text': None, 'stack_info': None, 'lineno': 18, 'funcName': '<module>', 'created': 1679061949.703206, 'msecs': 703.0, 'relativeCreated': 2.518892288208008, 'thread': 31644, 'threadName': 'MainThread', 'processName': 'MainProcess', 'process': 29348, 'message': 'Mensaje a nivel critical'}sc(time)s: CRITICAL [logger.py: 18] Mensaje a nivel critical
11:08:13 AM: DEBUG [logger.py:14] Mensaje a nivel debug
11:08:13 AM: INFO [logger.py:15] Mensaje a nivel info
11:08:13 AM: WARNING [logger.py:16] Mensaje a nivel warning
11:08:13 AM: ERROR [logger.py:17] Mensaje a nivel error
11:08:13 AM: CRITICAL [logger.py:18] Mensaje a nivel critical
04:11:21 PM: DEBUG [DBConnection.py:27] Connected successfully to DB: <connection object at 0x00000249BA8D28A0; dsn: 'user=postgres password=xxx dbname=Personas host=localhost port=5432', closed: 0>
04:11:21 PM: DEBUG [DBConnection.py:40] Cursor opened correctly <cursor object at 0x00000249BC3F83C0; closed: 0>
04:11:32 PM: DEBUG [DBConnection.py:27] Connected successfully to DB: <connection object at 0x000002321A22E8A0; dsn: 'user=postgres password=xxx dbname=Personas host=localhost port=5432', closed: 0>
04:11:32 PM: DEBUG [DBConnection.py:40] Cursor opened correctly <cursor object at 0x000002321A27BD80; closed: 0>
04:11:32 PM: DEBUG [DAO_Product.py:23] [<BusinessLogic.Product.Product object at 0x000002321BD36190>, <BusinessLogic.Product.Product object at 0x000002321BD36110>, <BusinessLogic.Product.Product object at 0x000002321BD36090>]
04:12:37 PM: DEBUG [DBConnection.py:27] Connected successfully to DB: <connection object at 0x00000205C07EE8A0; dsn: 'user=postgres password=xxx dbname=Personas host=localhost port=5432', closed: 0>
04:12:37 PM: DEBUG [DBConnection.py:40] Cursor opened correctly <cursor object at 0x00000205C2348040; closed: 0>
04:12:37 PM: DEBUG [DAO_Product.py:23] [<BusinessLogic.Product.Product object at 0x00000205C2345D50>, <BusinessLogic.Product.Product object at 0x00000205C2345F90>, <BusinessLogic.Product.Product object at 0x00000205C2345F50>]
04:14:47 PM: DEBUG [DBConnection.py:27] Connected successfully to DB: <connection object at 0x0000023430A4E8A0; dsn: 'user=postgres password=xxx dbname=Personas host=localhost port=5432', closed: 0>
04:14:47 PM: DEBUG [DBConnection.py:40] Cursor opened correctly <cursor object at 0x0000023430A9BD80; closed: 0>
04:14:47 PM: DEBUG [DAO_Product.py:23] [<BusinessLogic.Product.Product object at 0x00000234325560D0>, <BusinessLogic.Product.Product object at 0x0000023432556050>, <BusinessLogic.Product.Product object at 0x0000023432555FD0>]
04:15:23 PM: DEBUG [DBConnection.py:27] Connected successfully to DB: <connection object at 0x000001E9ABB0E8A0; dsn: 'user=postgres password=xxx dbname=Personas host=localhost port=5432', closed: 0>
04:15:23 PM: DEBUG [DBConnection.py:40] Cursor opened correctly <cursor object at 0x000001E9ABB5BD80; closed: 0>
04:15:23 PM: DEBUG [DAO_Product.py:23] [<BusinessLogic.Product.Product object at 0x000001E9AD638F50>, <BusinessLogic.Product.Product object at 0x000001E9AD638F90>, <BusinessLogic.Product.Product object at 0x000001E9AD639110>]
04:37:25 PM: DEBUG [DBConnection.py:27] Connected successfully to DB: <connection object at 0x0000022D2DD6E8A0; dsn: 'user=postgres password=xxx dbname=Personas host=localhost port=5432', closed: 0>
04:37:25 PM: DEBUG [DBConnection.py:40] Cursor opened correctly <cursor object at 0x0000022D2DDBBD80; closed: 0>
04:37:25 PM: DEBUG [DAO_Product.py:22] [<BusinessLogic.Product.Product object at 0x0000022D2F8A60D0>, <BusinessLogic.Product.Product object at 0x0000022D2F8A6050>, <BusinessLogic.Product.Product object at 0x0000022D2F8A6290>]
01:59:16 AM: DEBUG [DBConnection.py:27] Connected successfully to DB: <connection object at 0x0000016D32DF9CF0; dsn: 'user=postgres password=xxx dbname=Personas host=localhost port=5432', closed: 0>
01:59:16 AM: DEBUG [DBConnection.py:40] Cursor opened correctly <cursor object at 0x0000016D32E12DC0; closed: 0>
02:13:37 AM: DEBUG [DBConnection.py:27] Connected successfully to DB: <connection object at 0x000001ECC37EB010; dsn: 'user=postgres password=xxx dbname=Personas host=localhost port=5432', closed: 0>
02:13:37 AM: DEBUG [DBConnection.py:40] Cursor opened correctly <cursor object at 0x000001ECC3802DC0; closed: 0>
02:15:07 AM: DEBUG [DBConnection.py:27] Connected successfully to DB: <connection object at 0x000002128232E8A0; dsn: 'user=postgres password=xxx dbname=Personas host=localhost port=5432', closed: 0>
02:15:07 AM: DEBUG [DBConnection.py:40] Cursor opened correctly <cursor object at 0x000002128237BA00; closed: 0>
02:15:11 AM: DEBUG [DBConnection.py:27] Connected successfully to DB: <connection object at 0x000001C44D2FF9A0; dsn: 'user=postgres password=xxx dbname=Personas host=localhost port=5432', closed: 0>
02:15:11 AM: DEBUG [DBConnection.py:40] Cursor opened correctly <cursor object at 0x000001C44D28B5A0; closed: 0>
02:16:14 AM: DEBUG [DBConnection.py:27] Connected successfully to DB: <connection object at 0x0000029878ACF9A0; dsn: 'user=postgres password=xxx dbname=Personas host=localhost port=5432', closed: 0>
02:16:14 AM: DEBUG [DBConnection.py:40] Cursor opened correctly <cursor object at 0x0000029878A5BAE0; closed: 0>
02:20:00 AM: DEBUG [DBConnection.py:27] Connected successfully to DB: <connection object at 0x0000026C448AF9A0; dsn: 'user=postgres password=xxx dbname=Personas host=localhost port=5432', closed: 0>
02:20:00 AM: DEBUG [DBConnection.py:40] Cursor opened correctly <cursor object at 0x0000026C4483BAE0; closed: 0>
02:26:55 AM: DEBUG [DBConnection.py:27] Connected successfully to DB: <connection object at 0x00000165414CF9A0; dsn: 'user=postgres password=xxx dbname=Personas host=localhost port=5432', closed: 0>
02:26:55 AM: DEBUG [DBConnection.py:40] Cursor opened correctly <cursor object at 0x000001654145BCA0; closed: 0>
02:28:22 AM: DEBUG [DBConnection.py:27] Connected successfully to DB: <connection object at 0x000002406334F9A0; dsn: 'user=postgres password=xxx dbname=Personas host=localhost port=5432', closed: 0>
02:28:22 AM: DEBUG [DBConnection.py:40] Cursor opened correctly <cursor object at 0x00000240632DB5A0; closed: 0>
02:35:37 AM: DEBUG [DBConnection.py:27] Connected successfully to DB: <connection object at 0x000001CAD371FAB0; dsn: 'user=postgres password=xxx dbname=Personas host=localhost port=5432', closed: 0>
02:35:37 AM: DEBUG [DBConnection.py:40] Cursor opened correctly <cursor object at 0x000001CAD36AB680; closed: 0>
02:37:25 AM: DEBUG [DBConnection.py:27] Connected successfully to DB: <connection object at 0x0000021C9CFEB120; dsn: 'user=postgres password=xxx dbname=Personas host=localhost port=5432', closed: 0>
02:37:25 AM: DEBUG [DBConnection.py:40] Cursor opened correctly <cursor object at 0x0000021C9D002DC0; closed: 0>
02:40:49 AM: DEBUG [DBConnection.py:27] Connected successfully to DB: <connection object at 0x000001568D8EB120; dsn: 'user=postgres password=xxx dbname=Personas host=localhost port=5432', closed: 0>
02:40:49 AM: DEBUG [DBConnection.py:40] Cursor opened correctly <cursor object at 0x000001568D902DC0; closed: 0>
02:46:31 AM: DEBUG [DBConnection.py:27] Connected successfully to DB: <connection object at 0x00000288229DB120; dsn: 'user=postgres password=xxx dbname=Personas host=localhost port=5432', closed: 0>
02:46:31 AM: DEBUG [DBConnection.py:40] Cursor opened correctly <cursor object at 0x00000288229F2F80; closed: 0>
02:48:56 AM: DEBUG [DBConnection.py:27] Connected successfully to DB: <connection object at 0x0000021A88ADB120; dsn: 'user=postgres password=xxx dbname=Personas host=localhost port=5432', closed: 0>
02:48:56 AM: DEBUG [DBConnection.py:40] Cursor opened correctly <cursor object at 0x0000021A88AF2C00; closed: 0>
03:00:15 AM: DEBUG [DBConnection.py:27] Connected successfully to DB: <connection object at 0x0000018A7EBAB230; dsn: 'user=postgres password=xxx dbname=Personas host=localhost port=5432', closed: 0>
03:00:15 AM: DEBUG [DBConnection.py:40] Cursor opened correctly <cursor object at 0x0000018A7EBC2F80; closed: 0>
03:02:17 AM: DEBUG [DBConnection.py:27] Connected successfully to DB: <connection object at 0x000001C3D215B120; dsn: 'user=postgres password=xxx dbname=Personas host=localhost port=5432', closed: 0>
03:02:17 AM: DEBUG [DBConnection.py:40] Cursor opened correctly <cursor object at 0x000001C3D2172DC0; closed: 0>
